packages/components/src/components/tag/tag-utils.ts

```diff
@@ -1,11 +1,12 @@
 import { TAG_DISMISSIBLE_COLORS } from '../tag-dismissible/tag-dismissible-utils';
 import type { Theme } from '../../types';
-import { isThemeDark } from '../../utils/theme/isThemeDark'; // deep import needed since barrel contains MutationObserver and causes VRT to fail because of TAG_COLORS import
+import { isThemeDark } from '../../utils/theme/isThemeDark';
+import { ThemedColors } from '../../styles'; // deep import needed since barrel contains MutationObserver and causes VRT to fail because of TAG_COLORS import
```

Comment:
since it's a type import, it should be declared as such.

Reply:
I think comment is on the wrong line now?

Reply:
it's the right line, `ThemedColors` is a type

Reply:
Fixed it